front/app/containers/Admin/projects/project/timeline/edit.tsx

```diff
@@ -175,172 +128,129 @@ class AdminProjectTimelineEdit extends PureComponent<
           })
         )
         .subscribe((phaseFiles) => {
-          this.setState({ inStatePhaseFiles: phaseFiles as FileType[] });
+          setInStatePhaseFiles(phaseFiles as FileType[]);
         }),
-    ];
-  }
-
-  componentDidUpdate() {
-    const { projectId, id } = this.props.params;
-    this.params$.next({ projectId, id });
-  }
-
-  componentWillUnmount() {
-    this.subscriptions.forEach((subscription) => subscription.unsubscribe());
-  }
-
-  handleTitleMultilocOnChange = (title_multiloc: Multiloc) => {
-    this.setState(({ attributeDiff }) => ({
-      submitState: 'enabled',
-      attributeDiff: {
-        ...attributeDiff,
-        title_multiloc,
-      },
-    }));
-  };
+    ]);
 
-  handleEditorOnChange = (description_multiloc: Multiloc) => {
-    this.setState(({ attributeDiff }) => ({
-      submitState: 'enabled',
-      attributeDiff: {
-        ...attributeDiff,
-        description_multiloc,
-      },
-    }));
+    return () => {
+      subscriptions.forEach((subscription) => subscription.unsubscribe());
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please do not use this disable comment unless absolutely necessary, it's a very exceptional escape hatch.